src/pages/index.js

```diff
@@ -60,16 +165,71 @@ const profilePopup = new PopupWithForm({
 
     formData.input2 = formData.profile_description;
     delete formData.profile_description;
-
-    profileUserData.setUserInfo(formData);
+    api
+      .updateProfileInfo({ name: formData.input1, about: formData.input2 })
+      .then(
+        api.getUserInfo().then((formData) => {
```

Comment:
Please, don't call `api.getUserInfo()` when you edit the profile data, avatar or anything else because any profile updating method receives a **response** with the full profile data including the avatar